### DISCUSSION TITLE
Implementing UWB Pozyx DWM1000

### DISCUSSION THREAD

**Discussion by marcovilla118** [HefnySco/OBAL#20] [Q&A]
Hi,

first of all thank you so much for your job!!!

I would like to ask you if you have ever implemented the non-GPS positioning? I would like to implement Pozyx (with DWM1000) for UWB non gps positioning. 
Do you have any suggestion how to proceed? Electrical connection should be easy connecting the DWM1000 to the SPI of Raspberry but then there is for sure some firmware modifications (of course it's needed to prepare the anchor as well which can run on others raspberry as base stations).

Thanks and best regards

**Comment by HefnySco** [comment]
Very interesting ... SPI is used by IMU, in case you use OBAL-GY91 then SPI_CS0,1 are both used for IMU & Baro.

I suggest you use OBAL v1 the formal one. and then use SPI_CS1 to access DWM1000 on SPI.

Another solution which is harder to implement but not impossible is to reassign the GPIO ports to use the other SPI bus.

Please share your thoughts and experience when possible.

**Comment by HefnySco** [comment]
Hi,
I have a simple solution for you.
You can use another board for examlpe arduino nano to interface with this sensor then use a serial link to RPI-zero with mavlink message to OBAL.
